src/Document.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Ipssi\Evaluation;
+
+// Classe permettant de construire le doc à l'aide des éléments passés
+class Document
+{
+
+	private $name;
+	private $color;
+	private $content=[];
+
+	public function __construct($name, Color $color)
+	{
+		$this->name = $name;
+		$this->color = $color;
+	}
+
+	public function getName()
+	{
+		return $this->name;
+	}
+
+	public function addElement($object)
```

Comment:
```suggestion
	public function addElement(Element $object)
```
Il fallait créer une class Element